docs/devenv_rhel8.md

```diff
@@ -76,18 +89,20 @@ packaging/rpm/_rpmbuild/SRPMS/microshift-4.10.0-nightly_1654189204_34_gc871db21.
 
 ## Run MicroShift Executable
 Log into the development virtual machine with the `microshift` user credentials.
+
 ### Installing Clients
 Run the following commands to install `oc` and `kubectl` utilities.
 ```bash
 curl -O https://mirror.openshift.com/pub/openshift-v4/$(uname -m)/clients/ocp/stable/openshift-client-linux.tar.gz
 sudo tar -xf openshift-client-linux.tar.gz -C /usr/local/bin oc kubectl
 rm -f openshift-client-linux.tar.gz
 ```
+
 ### Runtime Prerequisites
 Run the following commands to install CRI-O.
 ```bash
-sudo subscription-manager repos --enable rhocp-4.10-for-rhel-8-$(uname -i)-rpms fast-datapath-for-rhel-8-$(uname -i)-rpms
```

Comment:
Why are we removing the FDP channel? It's still necessary

Reply:
Ahhh it's down on the other section

Reply:
Enabling two repos in one line didn't work for me :-)
I had to enable one repo at a time, then the other.

Reply:
Yes, this is because --enable should have been specified before each repo. There was a problem in the previous documentation.